src/main/java/core/basesyntax/DiscountStrategy.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import core.basesyntax.strategy.DiscountService;
+import core.basesyntax.strategy.impl.BirthdayDiscountService;
+import core.basesyntax.strategy.impl.BlackFridayDiscountService;
+import core.basesyntax.strategy.impl.DefaultDiscountService;
+import core.basesyntax.strategy.impl.NewYearDiscountService;
+
+public class DiscountStrategy {
+    public static final String BIRTHDAY = "Birthday";
+    public static final String BLACK_FRIDAY = "Black Friday";
+    public static final String NEW_YEAR = "New Year";
```

Comment:
[Do not create constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/patterns/strategy#do-not-create-constants-for-the-discount-type-and-discount-amount)